app/team.go

```diff
@@ -430,6 +434,9 @@ func (a *App) joinUserToTeam(team *model.Team, user *model.User) (*model.TeamMem
 }
 
 func (a *App) JoinUserToTeam(team *model.Team, user *model.User, userRequestorId string) *model.AppError {
+	if !a.isTeamEmailAllowed(user, team) {
+		return model.NewAppError("isTeamEmailAllowed", "api.team.is_team_creation_allowed.domain.app_error", nil, "", http.StatusBadRequest)
```

Comment:
The convention here is to name the function in which it failed, e.g. passing `JoinUserToTeam` as the first parameter.

Reply:
Fixed.